composer.json

```diff
@@ -83,6 +88,7 @@
         "illuminate/view": "self.version"
     },
     "require-dev": {
+        "ext-gmp": "*",
```

Comment:
I think we still need this in `require-dev`.

See this PR: https://github.com/laravel/framework/pull/43959

Seems best to leave it as GMP objects have some quirks that this addresses and I can't replicate with standard classes.